### PR TITLE
chore: Added description for rule 942450

### PR DESCRIPTION
**Pull request by Xhoenix**
Description for SQLi hex evasion detection rule

### CODE REVIEW

File: rules/REQUEST-942-APPLICATION-ATTACK-SQLI.conf [hunk 1]
```diff
@@ -1368,6 +1368,9 @@ SecRule REQUEST_COOKIES|!REQUEST_COOKIES:/__utm/|!REQUEST_COOKIES:/_pk_ref/|REQU
 #
 # -=[ SQL Hex Evasion Methods ]=-
 #
+# Hex encoding detection:
+# (?i:\b0x[a-f\d]{3,}) #match any 3 or more hex bytes after "0x", together forming a hexadecimal payload(e.g 0xf00, 0xf00d and so on)
```
Comment (theseion) [inline rules/REQUEST-942-APPLICATION-ATTACK-SQLI.conf:1372]:
```suggestion
# (?i:\b0x[a-f\d]{3,}) will match any 3 or more hex bytes after "0x", together forming a hexadecimal payload(e.g 0xf00, 0xf00d and so on)
```

**Review by theseion — CHANGES_REQUESTED** [review]

**Review by theseion — APPROVED** [review]

**Comment by theseion** [comment]
Thanks @Xhoenix.

**Comment by dune73** [comment]
Good stuff.

**Comment by RedXanadu** [comment]
Sorry for the delay, I didn't get around to merging this one in time as promised :laughing:

@Xhoenix Was this your first PR? If so, we'll need to add you to the list of contributors :tada:

**Comment by dune73** [comment]
Far from it.

```
$ git log | grep -e "Merge pull request.*Xhoenix"
    Merge pull request #3219 from Xhoenix/v4.0/dev
    Merge pull request #3215 from Xhoenix/v4.0/dev
    Merge pull request #3214 from Xhoenix/v4.0/dev
    Merge pull request #3213 from Xhoenix/v4.0/dev
    Merge pull request #3212 from Xhoenix/v4.0/dev
```

**Pull request merged**